### PR TITLE
custom templates

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony @smn ready for review. You can pay less attention to the CSS and HTML changes (I imagine this will change again when the designs are finalized). Python code and Jinja macros are worth reviewing though.

File: springboard_iogt/application.py [hunk 1]
```diff
@@ -12,7 +12,8 @@ def main(global_config, **settings):
     defaults = dict(cp.items('springboard:pyramid'))
     defaults['jinja2.filters'] += (
         '\nrecent_pages = springboard_iogt.filters:recent_pages'
-        '\ncategory_dict = springboard_iogt.filters:category_dict')
+        '\ncategory_dict = springboard_iogt.filters:category_dict\n'
+        'content_section = springboard_iogt.filters:content_section')
```
Comment (smn) [inline springboard_iogt/application.py]: I should've probably gone with a `'\n'.join(list)` here in the scaffolded app.

Comment (Rizziepit) [inline springboard_iogt/application.py]: I've made this a bit neater now

**Comment by miltontony** [comment]
:+1:

**Pull request merged**